### ISSUE TITLE
SerializableStackTraceElement is hard to read in the logs

### ISSUE THREAD

**Issue by a-k-g** [palantir/conjure-java-runtime#347]
When a RemoteException is logged, it prints out the `SerializableError remoteException` with a simple `toString()`. The `SerializableError` has a custom `toString()` which nicely prints out each individual `SerializableStackTraceElement` in the remote stacktrace. However, the `SerializableStackTraceElement` doesn't have a `toString()` and uses the default one generated by Immutables. This results in a hard to read log which looks something like -

```
Remote stacktrace:
	at SerializableStackTraceElement{className=com.palantir.remoting.http.errors.SerializableErrorToExceptionConverter, methodName=getException, fileName=SerializableErrorToExceptionConverter.java, lineNumber=82}
	at SerializableStackTraceElement{className=com.palantir.remoting.http.errors.FeignSerializableErrorErrorDecoder, methodName=decode, fileName=FeignSerializableErrorErrorDecoder.java, lineNumber=43}
	at SerializableStackTraceElement{className=feign.SynchronousMethodHandler, methodName=executeAndDecode, fileName=SynchronousMethodHandler.java, lineNumber=126}
	at SerializableStackTraceElement{className=feign.SynchronousMethodHandler, methodName=invoke, fileName=SynchronousMethodHandler.java, lineNumber=74}
	at SerializableStackTraceElement{className=feign.ReflectiveFeign$FeignInvocationHandler, methodName=invoke, fileName=ReflectiveFeign.java, lineNumber=103}
	at SerializableStackTraceElement{className=com.sun.proxy.$Proxy55, methodName=openFile, lineNumber=-1}
	at SerializableStackTraceElement{className=sun.reflect.NativeMethodAccessorImpl, methodName=invoke, fileName=NativeMethodAccessorImpl.java, lineNumber=62}
	at SerializableStackTraceElement{className=sun.reflect.DelegatingMethodAccessorImpl, methodName=invoke, fileName=DelegatingMethodAccessorImpl.java, lineNumber=43}
	at SerializableStackTraceElement{className=java.lang.reflect.Method, methodName=invoke, fileName=Method.java, lineNumber=498}
	at SerializableStackTraceElement{className=com.palantir.tritium.proxy.InvocationEventProxy, methodName=execute, fileName=InvocationEventProxy.java, lineNumber=111}
	at SerializableStackTraceElement{className=com.palantir.tritium.proxy.InvocationEventProxy, methodName=instrumentInvocation, fileName=InvocationEventProxy.java, lineNumber=88}
	at SerializableStackTraceElement{className=com.palantir.tritium.proxy.InvocationEventProxy, methodName=handleInvocation, fileName=InvocationEventProxy.java, lineNumber=69}
	at SerializableStackTraceElement{className=com.google.common.reflect.AbstractInvocationHandler, methodName=invoke, fileName=AbstractInvocationHandler.java, lineNumber=87}
	at SerializableStackTraceElement{className=com.sun.proxy.$Proxy62, methodName=putFiles, lineNumber=-1}
	at SerializableStackTraceElement{className=sun.reflect.NativeMethodAccessorImpl, methodName=invoke0, fileName=NativeMethodAccessorImpl.java, lineNumber=-2}
```

Would be great if the `SerializableStackTraceElement` could print out each element as a stacktrace is usually printed.

Happy to contribute a PR if people agree with this suggestion.

**Comment by uschi2000** [comment]
Yep, that works. Thanks!

**Comment by a-k-g** [comment]
Released in 2.0.0
